x-pack/docs/en/rest-api/security/get-service-accounts.asciidoc

```diff
@@ -84,6 +84,20 @@ GET /_security/service/elastic/fleet-server
           ],
           "allow_restricted_indices": false
         },
+        {
+          "names" : [
+            "traces-apm.sampled-*"
```

Comment:
Is the data going into different namespaces or is it just `default`. Would be great if we can narrow it down further.

Reply:
It'll be whatever namespace is configured in the policy, so not limited to `default`.

Reply:
Ok, so it is only fleet-server that has access to all the indices but the API that would be handed out would be specific to a data stream?

Reply:
Yes